bedrock/mozorg/templates/mozorg/contribute/tasks/v1/build-firefox.html

```diff
@@ -1,28 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm not sure what's happening with the experiment but I removed the links to these pages from their respective signup pages. Might it be worth adding a redirect? Or maybe a 404 is the best way to treat it after all?